src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.figuresupplier.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier1 = new FigureSupplier();
+        Drawable circle = new Circle("red", 21);
```

Comment:
these figures must definitely have the type Drawable?

Reply:
And you don't need create figure manual, its do method getRandomFigure or getDefaultFigure

Reply:
1. I think it's a good tone when you use the methods assigned to the object through the interface
2. In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters.